.changeset/eleven-otters-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+'style-dictionary': patch
```

Comment:
As far as I understood, the examples are published to NPM so users can use `style-dictionary init` to initialize using the examples, which would warrant a patch to get it released right?

Reply:
I wouldn't mind putting them as workspaces in a monorepo but then the package name under which the examples are published would be different, meaning that for it to work with `style-dictionary init`, the examples would either need to be peer dependencies or something, or the style-dictionary init CLI would be attached to the examples as a dev depenedencies, so that they can install "@style-dictionary/example-foo" which has that init CLI as a dependency and then run it like `npx style-dictionary-init foo` or something. Food for thought for sure, but a breaking change no matter what I think so maybe we should plan that for v5 in the long-term future